### PR TITLE
Use the same command line parser as LPython and LFortran

### LINKED ISSUE #35: Use the same command line parser as LPython and LFortran
And make the main driver similar. Possibly extract common parts into libasr.

### PR DESCRIPTION
**Pull request by Shaikh-Ubaid**
fixes #35

### CODE REVIEW

File: integration_tests/CMakeLists.txt [hunk 1]
```diff
@@ -179,49 +179,28 @@ RUN(NAME expr1.c LABELS gcc c wasm llvm NOFAST)
...
-RUN(NAME array_18.cpp LABELS gcc llvm NOFAST
-    EXTRA_ARGS --extra-arg=-I${CONDA_PREFIX}/include)
-RUN(NAME array_19.cpp LABELS gcc llvm NOFAST
-    EXTRA_ARGS --extra-arg=-I${CONDA_PREFIX}/include)
-RUN(NAME array_20.cpp LABELS gcc llvm NOFAST
-    EXTRA_ARGS --extra-arg=-I${CONDA_PREFIX}/include)
-RUN(NAME array_21.cpp LABELS gcc llvm NOFAST
-    EXTRA_ARGS --extra-arg=-I${CONDA_PREFIX}/include)
-RUN(NAME array_22.cpp LABELS gcc llvm NOFAST
-    EXTRA_ARGS --extra-arg=-I${CONDA_PREFIX}/include)
```
Comment (Shaikh-Ubaid) [inline integration_tests/CMakeLists.txt:227]: The above test cases are intact. Just the `EXTRA_ARGS` is removed.

**Comment by Shaikh-Ubaid** [comment]
The reference tests simply got renamed.

**Comment by Shaikh-Ubaid** [comment]
Ready.

**Review by certik — APPROVED** [review]
Awesome, thanks! I think this looks great and good to merge. We can iron out any regressions later.

Go ahead and merge it.

**Pull request merged**